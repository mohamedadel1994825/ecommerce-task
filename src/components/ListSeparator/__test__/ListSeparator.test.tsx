import { render } from "@testing-library/react-native";
import React from "react";
import { Separator } from "../index";

describe("Separator", () => {
  it("should match snapshot", () => {
    const { toJSON } = render(<Separator />);
    expect(toJSON()).toMatchSnapshot();
  });
  it("should render correctly", () => {
    const { getByTestId } = render(<Separator />);
    const separator = getByTestId("list-item-separator");
    expect(separator).toBeDefined();
  });
});
