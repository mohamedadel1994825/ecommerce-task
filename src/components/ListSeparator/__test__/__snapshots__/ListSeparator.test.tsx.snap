// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Separator should match snapshot 1`] = `
<View
  style={
    {
      "width": 15,
    }
  }
  testID="list-item-separator"
/>
`;
